tests/e2e/framework/framework.go

```diff
@@ -84,13 +84,15 @@ func InitLogging() error {
 	return nil
 }
 
-// NewTestConfig creates a full config will all supported configs.
-func NewTestConfig(testID, baseVersion string) (*CommonConfig, error) {
+// NewCommonConfigWithBaseVersion creates a new CommonConfig with the specified
+// version of Istio. If baseVersion is empty, it will use the local head
```

Comment:
nit: baseVersion -> version to be consistent with function param

Reply:
I was referring to the function comments tho. Updating the function name is also superb.